web/ui/react-app/src/vendor/flot/jquery.flot.selection.js

```diff
@@ -0,0 +1,414 @@
+/**
+ *
+ * THIS FILE WAS COPIED INTO PROMETHEUS FROM GRAFANA'S VENDORED FORK OF FLOT
+ * (LIVING AT https://github.com/grafana/grafana/tree/v7.5.8/public/vendor/flot),
+ * WHICH CONTAINS FIXES FOR DISPLAYING NULL VALUES IN STACKED GRAPHS. THE ORIGINAL
+ * FLOT CODE WAS LICENSED UNDER THE MIT LICENSE AS STATED BELOW. ADDITIONAL
+ * CHANGES HAVE BEEN CONTRIBUTED TO THE GRAFANA FORK UNDER AN APACHE 2 LICENSE, SEE
+ * https://github.com/grafana/grafana/blob/v7.5.8/LICENSE.
```

Comment:
I added the `v7.5.8` distinction as that was the last version that was Apache 2 licensed and the one I got this file from.

Reply:
I copied the header from another file and I just noticed it says `CONTAINS FIXES FOR DISPLAYING NULL VALUES IN STACKED GRAPHS`, which has nothing to do with this plugin, but I checked and all the other plugins have it. Maybe we should change that?

Reply:
We should update the license in other files :(

Reply:
@juliusv Do you know which one this actually pertains to?

Reply:
Nope! So https://github.com/prometheus/prometheus/pull/6603 just copied Grafana's vendored flot version into Prometheus. The reason for this was that the normal Flot version didn't handle null values in stacked graphs correctly, but Grafana's fork did, because they did something to it to fix it. So I guess one would have to do some archaeological digging in Grafana's Flot fork itself to try and find out what they changed, but not sure how easy it is. Also, who knows, maybe upstream Flot has been fixed wrt. stacked null values in the meantime.

Reply:
I'll remove that line from this new file because we definitely know it has nothing to do with the fix. Doing updates on the other files/maybe using the new version of upstream Flot is a separate PR.

Reply:
It may also be good to add the `v7.5.8` tag to the other links, especially the ones referring to the Apache 2 license that is now not an Apache 2 license.